### PR TITLE
Cfy 4382 sanity checks

### PR DESCRIPTION
**Pull request by konradxyz**

### CODE REVIEW

File: resources/rest-service/cloudify/types/types.yaml [hunk 1]
```diff
@@ -178,6 +178,25 @@ node_types:
               default: 300
 
 
+        #####################################################
+        # This operation attempts to communicate with agent
```
Comment (dankilman) [inline resources/rest-service/cloudify/types/types.yaml]: with agent -> with an agent

File: resources/rest-service/cloudify/types/types.yaml [hunk 2]
```diff
@@ -178,6 +178,25 @@ node_types:
...
+        # that has already been installed. This operation
+        # should not have side effects.
+        # ---------------------------------------------------
+        # It will be executed on central deployment agent.
```
Comment (dankilman) [inline resources/rest-service/cloudify/types/types.yaml]: on central -> on the central

File: resources/rest-service/cloudify/types/types.yaml [hunk 3]
```diff
@@ -178,6 +178,25 @@ node_types:
...
+        # This request might be sent to other manager's
```
Comment (dankilman) [inline resources/rest-service/cloudify/types/types.yaml]: to other manager's rabbitmq server. -> to a different manager's RabbitMQ server

**Comment by pleszcz** [comment]
@dankilman What's wrong with the Travis now? I've changed the comments only. Error (taken from the Travis log):

```
======================================================================
FAIL: test_policies_flow_with_diamond (workflow_tests.test_policies.TestPolicies)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/cloudify-cosmo/cloudify-manager/tests/workflow_tests/test_policies.py", line 114, in test_policies_flow_with_diamond
    self.wait_for_executions(self.NUM_OF_INITIAL_WORKFLOWS + 1)
  File "/home/travis/build/cloudify-cosmo/cloudify-manager/tests/workflow_tests/test_policies.py", line 60, in wait_for_executions
    self.do_assertions(assertion)
  File "/home/travis/build/cloudify-cosmo/cloudify-manager/tests/testenv/__init__.py", line 140, in do_assertions
    **kwargs)
  File "/home/travis/build/cloudify-cosmo/cloudify-manager/tests/testenv/utils.py", line 253, in do_retries
    return func(**kwargs)
  File "/home/travis/build/cloudify-cosmo/cloudify-manager/tests/workflow_tests/test_policies.py", line 59, in assertion
    self.assertEqual(expected_count, len(executions))
AssertionError: 3 != 2

----------------------------------------------------------------------
```

**Comment by dankilman** [comment]
You can safely ignore, specifically this failure, it is a sporadic failure that started showing up which will get fixed some time soon.

**Comment by pleszcz** [comment]
@dankilman Please close this pull request. New one is #375

**Pull request closed without merging**